lib/pipeline.js

```diff
@@ -75,30 +81,32 @@ class PipelineModel extends BaseModel {
             toUnarchive: []
         };
         const existingPRs = existingJobs.filter(j => j.isPR());   // list of PRs according to SD
-        const existingPRsNames = existingPRs.map(j => j.name);
         const openedPRsNames = openedPRs.map(j => j.name);
         const openedPRsRef = openedPRs.map(j => j.ref);
 
         existingPRs.forEach((job) => {
+            // getting PR and number e.g. PR-1
+            const prName = job.name.match(PR_JOB_NAME)[PR_NAME_GROUP.pr];
```

Comment:
abstract this behind a function `getNameFromJob(job, type)` where type can be `pr` or `job`
```
function getNameFromJob(job, type) {
    return job.name.match(PR_JOB_NAME)[REGEX_CAPTURING_GROUP[type]];
}
```